src/mappings/factoryRouter.ts

```diff
@@ -102,33 +95,6 @@ export function handleTokenRemoved(event: TokenRemoved): void {
   opc.save()
 }
 
-export function handleSSContractAdded(event: SSContractAdded): void {
```

Comment:
we don't need to remove handleSSContractAdded and handleSSContractRemoved

Reply:
I removed it as we don't have a mapping for SSContract anymore. Without a mapping, the graph doesn't generate any types for SSContract and `SSContract.create(event.params.contractAddress)` breaks

Reply:
hmm, then we need to add them back, because they could possibly make sense in the future. In theory we can have multiple ss contracts.

Reply:
`handleSSContractAdded` and `handleSSContractRemoved` are back